src/app/store/app-store.module.ts

```diff
@@ -12,27 +12,30 @@ import { UserStateModule } from './user/user-state.module';
 import { SpinnerStateModule } from '@store/spinner/spinner-state.module';
 import { ReferenceDataStateModule } from './reference-data/reference-data.module';
 import { TestStationsStateModule } from './test-stations/test-stations-state.module';
+import { DefectsStateModule } from './defects/defects-state.module';
 
 @NgModule({
   declarations: [],
   imports: [
-    CommonModule,
```

Comment:
Why the re-ordering?

Reply:
Autogenerated components are added in alphabetical order by the CLI, as I was inserting a new module here, I thought I'd replicate that for cleaner code.

Reply:
This new order breaks the dev tools and potentially other modules. at the very least the first few modules should be : `CommonModule, StoreModule.forRoot({}), EffectsModule.forRoot([]) and then StoreDevtoolsModule` in that order. Can't use dev tools before store is bootstrapped.

Reply:
Suggestion applied.